bin/add_callers_to_scout_custom.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python3
+
+import argparse
+from collections import defaultdict
+
+
+def parse_arguments():
+    """Parse command-line arguments."""
+    parser = argparse.ArgumentParser(
+        description="Copy caller names from SVDB INFO.set to INFO.SCOUT_CUSTOM in SVDB merged VCFs"
+    )
+    parser.add_argument(
+        "--merged_vcf", required=True, help="Path to the merged VCF file."
+    )
+    parser.add_argument(
+        "--callers", required=True, help="Comma-separated list of used callers."
+    )
+    return parser.parse_args()
+
+
+def process_vcf(merged_vcf, used_callers):
```

Comment:
What about adding types? (Will only comment on this once in this PR).

This would also help guide my reviewing (i.e. by opening VSCode and looking for the red)